strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/MavenProxyRepositoryProviderTestIT.java

```diff
@@ -291,18 +227,34 @@ public void testMavenCentral(@MavenRepository(storageId = STORAGE_ID,
     @ExtendWith({ RepositoryManagementTestExecutionListener.class })
     @Test
     public void testStrongboxAtCarlspringDotOrg()
-            throws IOException
+            throws Exception
     {
-        RepositoryPath path = artifactResolutionService.resolvePath("public",
-                                                                    "maven-group",
-                                                                    "org/carlspring/commons/commons-io/1.0-SNAPSHOT/maven-metadata.xml");
-        try (InputStream is = artifactResolutionService.getInputStream(path))
-        {
+        assertStreamNotNull("public",
+                            "maven-group",
+                            "org/carlspring/commons/commons-io/1.0-SNAPSHOT/maven-metadata.xml");
+    }
+
+    private void assertStreamNotNull(final String storageId,
```

Comment:
@ptirador we probably should avoid copypasting this method within test classes

Reply:
see [comment](https://github.com/strongbox/strongbox/pull/1434/files#r317427139)